aggregator/demo/snapshot-only-plugins.txt

```diff
@@ -0,0 +1,3 @@
+pipeline-build-step
+pipeline-input-step
+pipeline-stage-step
```

Comment:
Yes! That is a key aspect of this change—the source location of plugins no longer matters for purposes of the aggregator or demo, so they could be moved into their own repositories, and there is no longer reduced support for plugins not in the original `org.jenkins-ci.plugins.workflow` group.